DOC-12 | Improve PRUNE docs / UDFs and DB-Server only functions not allowed in PRUNE (#1193)
arangodb/docs: 3.8/aql/extending.md

When used in clusters, UDFs are always executed on the
[Coordinator](../architecture-deployment-modes-cluster-architecture.html).

> Queries that would push UDF execution to DB-Servers, e.g. if a UDF is used 
> inside a traversal PRUNE condition, will abort with a parse error.

Author: ```suggestion
Queries that would push UDF execution to DB-Servers will abort with a parse error.
UDFs cannot be used in traversal PRUNE conditions at all.
```
